zulip_bots/zulip_bots/bots/lunch/lunch.py

```diff
@@ -0,0 +1,220 @@
+import random
+
+ORDERS = {}
+
+class LunchBotHandler(object):
+
+    def usage(self):
+        return '''
+        This is a small Zulip bot for organizing lunch with your friends and
+        coworkers.
+        '''
+
+    # MESSAGE RECEIVERS
+    def handle_message(self, message, bot_handler):
+        content = self.keyword_selection(message)
+        print(message)
+        bot_handler.send_reply(message, content)
```

Comment:
Could you please explain what's the purpose of the `print()` here? Was this just for debugging and perhaps you forgot to remove it? :)

Reply:
Ups, debugging :)